integration/js/utils/CloudWatch.js

```diff
@@ -4,13 +4,16 @@ var cloudWatch = new AWS.CloudWatch({
   apiVersion: '2010-08-01'
 });
 
-module.exports.emitMetric = async (namespace, capabilities, metric_name, value, namespaceInfix) => {
+// cwNamespaceInfix is a param that is used to update the namespace of CloudWatch metrics.
+// This infix will be added when this variable is passed to the KITE test from the payload.
+// It is used to differentiate metrics between the previous major release and the latest release.
```

Comment:
Hmm, I think the cwNamespaceInfix is self explanatory, but the larger problem is it's just not clear what the emitMetric function is supposed to do in the the first place. So if you could change the comment to be something more like `This function will create a metric in Cloudwatch using the given params in this format: TestType/cwNamespaceInfix/namespace/metric_name` helps a lot with understanding what the function is actually doing.

Reply:
Done